app/services/triannon/ldp_loader.rb

```diff
@@ -4,29 +4,28 @@ module Triannon
   # Loads an existing Annotation from the LDP server
   class LdpLoader
 
-
     # @param [String] id the unique id of the annotation.  Can include base_uri prefix or omit it.
-    def self.load id
-      l = Triannon::LdpLoader.new id
+    # @param [String] root_container the LDP parent container for the annotation
+    def self.load(id, root_container)
+      l = Triannon::LdpLoader.new(id, root_container)
       l.load_anno_container
       l.load_bodies
       l.load_targets
 
-      oa_graph = Triannon::LdpToOaMapper.ldp_to_oa l.ldp_annotation
+      oa_graph = Triannon::LdpToOaMapper.ldp_to_oa(l.ldp_annotation, root_container)
       oa_graph
     end
 
-    # @deprecated was needed by old annotations#index action, which now redirects to search (2015-04)
-    def self.find_all
-      l = Triannon::LdpLoader.new
-      l.find_all
-    end
-
     attr_accessor :ldp_annotation
 
     # @param [String] id the unique id of the annotation.  Can include base_uri prefix or omit it.
-    def initialize id = nil
+    # @param [String] root_container the LDP parent container for the annotation
+    def initialize(id = nil, root_container)
```

Comment:
As above, would it help to put the `root_container` before the `id`? At the very least, it seems a little strange to have an optional parameter before a required one.